cogs/misc.py

```diff
@@ -420,6 +429,15 @@ async def parse(self, ctx, *, time_and_arg: converters.HumanTime(other=True)):
                            f'Other argument: {time_and_arg.other}\n'
                            f'Delta: {(time - pendulum.instance(ctx.message.created_at)).in_words()}')
 
+    @commands.command()
+    async def someone(self, ctx, *, text)
```

Comment:
Totally missed it in the previous review, there is a syntax error on this line. Fix it